### ISSUE TITLE
Using external browser

### ISSUE THREAD

**Issue by Joly0** [wkaisertexas/tiktok-uploader#176]
Hey, i wanted to know if it is possible to use external browsers such as browserless-v2 https://docs.browserless.io/ ?

**Comment by wkaisertexas** [comment]
No, supporting browser-less is not supported and out of scope for this project.
